ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
This makes no sense. I can't be dependent on something only if I am doing a full build but not if I am only building the client tools.

I suspect what is really happening is that the client tool is dependent on the scm file of the service that it calls (which is fair enough).